packages/api/test/e2e/governance.e2e.ts

```diff
@@ -0,0 +1,168 @@
+
+import initApiPromise from '../../../../jest/initApiPromise';
+import { AccountId, Vec } from '@cennznet/types';
+import {DeriveProposalInfo} from "@cennznet/api/derives/governance/types";
+
+let api;
+const keyring = new Keyring({ type: 'sr25519' });
+let councilMemberAlice, councilMemberBob, councilMemberCharlie, councilMemberDave;
+
+beforeAll(async done => {
+  await cryptoWaitReady();
+  api = await initApiPromise();
+  const sudoKey = await api.query.sudo.key();
+  // alice is sudo
+  councilMemberAlice = keyring.addFromUri('//Alice');
+  councilMemberDave = keyring.addFromUri('//Dave');
+  councilMemberBob = keyring.addFromUri('//Bob');
+  councilMemberCharlie = keyring.addFromUri('//Charlie');
+  // Lookup from keyring (assuming we have added all, on --dev this would be `//Alice`)
+  const sudoPair = keyring.getPair(sudoKey.toString());
+
+  const transaction1 = api.tx.governance.addCouncilMember(councilMemberAlice.address);
+  const transaction2 = api.tx.governance.addCouncilMember(councilMemberBob.address);
+  const transaction3 = api.tx.governance.addCouncilMember(councilMemberCharlie.address);
+  const transaction4 = api.tx.governance.addCouncilMember(councilMemberDave.address);
+
+  const batchCouncilEx = api.tx.utility.batch([
+    transaction1,
+    transaction2,
+    transaction3,
+    transaction4
+  ]);
+
+  await api.tx.sudo
+    .sudo(batchCouncilEx)
+    .signAndSend(sudoPair, async ({ events, status }) => {
+      if (status.isInBlock) {
+        for (const {event: {method, section, data}} of events) {
+          console.log('Method:', method.toString());
+          console.log('section:', section.toString());
+        }
+        console.log(`Transaction included at blockHash ${status.asInBlock}`);
+        done();
+      }
+    });
+
+});
+
+afterAll(async () => {
+  await api.disconnect();
+});
+
+describe('Governance', () => {
+
+  it('List all council members', async done => {
+    const councilMembers: Vec<AccountId> = await api.query.governance.council();
+    console.log('Councilmemmbers::....', councilMembers.toJSON());
+    const members: string[] = councilMembers.toJSON() as unknown as string[];
+    expect(members.includes(councilMemberBob.address)).toBeTruthy();
+    expect(members.includes(councilMemberAlice.address)).toBeTruthy();
+    expect(members.includes(councilMemberCharlie.address)).toBeTruthy();
+    expect(members.includes(councilMemberDave.address)).toBeTruthy();
+    done();
+  });
+
+  it('creates a proposal to add new council member', async done => {
+    const newCouncilMember = '5FWEHQqYMN8YCg8yJxKHnon7Dtx4Psp2xnjvKfQqGC6kUwgv';
+    const call = api.tx.governance.addCouncilMember(newCouncilMember);
+    const proposalCall = api.registry.createType('Call', call).toHex();
+    const justificationUri = 'https://example.com/nft/metadata'
+    const enactmentDelay = 26; // execute after 26 blocks once sufficient votes criteria is met
+    await api.tx.governance.submitProposal(
```

Comment:
might be nice at some point if we have a wrapper method which includes adding council member and creating the proposalCall, so people just need to put in council members pub key.

Reply:
plan is initially the council members would be created using sudo, then sudo would be set to null.. meaning no sudo..
After this adding a new council member, will be done only via council member by submitProposal - proposal would be to add a new council member..

Reply:
in the UI it will look like
<img width="1008" alt="Screen Shot 2021-08-11 at 12 20 51 PM" src="https://user-images.githubusercontent.com/29415595/128951214-d4bee98e-6d07-4d87-bdd6-b39b574c9a5e.png">
so the call is not just about adding council member, it can be anything